config/initializers/assets.rb

```diff
@@ -1,2 +1,3 @@
 Rails.application.config.assets.version = "1.0"
 Rails.application.config.assets.precompile += %w( auth/cover.js )
+Rails.application.config.assets.precompile << /\.(?:woff2)\z/
```

Comment:
wouldn't it be easier to use globbing? ie `Rails.application.config.assets.precompile += %w( auth/cover.js *.woff2 )`